packages/select/src/components/select/multiSelect.tsx

```diff
@@ -184,11 +184,12 @@ export class MultiSelect<T> extends React.PureComponent<IMultiSelectProps<T>, IM
             if (this.input != null && this.input !== document.activeElement) {
                 // the input is no longer focused so we can close the popover
                 this.setState({ isOpen: false });
+                Utils.safeInvokeMember(this.props.popoverProps, "onInteraction", false);
             } else if (!this.props.openOnKeyDown) {
```

Comment:
separately, I don't understand what this code is doing.
`if (!this.props.openOnKeyDown)`, we always set `isOpen: true`, regardless of the value of `nextOpenState`. I don't understand that. I'd think this would make any Popover with `!this.props.openOnKeyDown` be impossible to close

Reply:
I'd love to change this code entirely to be:
```
if (this.input != null && this.input === document.activeElement) {
    // the input is active, we stay open regardless of `nextOpenState`
    this.setState({ isOpen: true });
    Utils.safeInvokeMember(this.props.popoverProps, "onInteraction", true);
} else {
    this.setState({ isOpen: false });
    Utils.safeInvokeMember(this.props.popoverProps, "onInteraction", false);
}
```

Reply:
that code looks good to me, and it seems to pass unit tests on my end, do you want to push it up or should I add a commit?